covid_xprize/examples/predictors/conditional_lstm/train_predictor.py

```diff
@@ -65,6 +66,7 @@ def train_predictor(training_data: pd.DataFrame,
     :param context_column: Which column in the data df to use as context and outcome.
     :param arch: Which predictor architecture to use.
         Current options are 'conditional' and 'independent'.
+    :param return_all_trials: If set to True, then this function returns all trials as a list.
```

Comment:
A "trial" is a trained model in that case? Maybe worth mentioning it's going to return "all trials as a list of trained models"

Reply:
👍